cli/pack/deb_control_dir.go

```diff
@@ -61,40 +80,21 @@ func initControlDir(destDirPath string, ctx *context.Ctx) error {
 		"Depends":      "",
 	}
 
+	var deps common.PackDependencies
 	if !ctx.Tarantool.TarantoolIsEnterprise {
-		minTarantoolVersion := ctx.Tarantool.TarantoolVersion
-		maxTarantoolVersion, err := common.GetNextMajorVersion(minTarantoolVersion)
-		if err != nil {
-			return project.InternalError("Failed to get next Tarantool major version: %s", err)
+		var err error
+		if deps, err = ctx.Pack.Deps.AddTarantool(ctx.Tarantool.TarantoolVersion); err != nil {
+			return fmt.Errorf("Failed to add tarantool dependency: %s", err)
 		}
+	}
 
-		addDependency(&debControlCtx, common.PackDependency{
-			Name: "tarantool",
-			Relations: []common.DepRelation{
-				{
-					Relation: ">=",
-					Version:  minTarantoolVersion,
-				},
-				{
-					Relation: "<<",
-					Version:  maxTarantoolVersion,
-				},
-			},
-		})
+	deps = append(deps, ctx.Pack.Deps...)
+	for _, dependency := range formatDeb(deps) {
+		addDependency(&debControlCtx, dependency)
```

Comment:
Why did you separated `formatDeb`? This logic can be placed directly in `addDependency` function. Moreover, it can be `addDependencies` function that hides a loop.